go/vt/schemadiff/schema.go

```diff
@@ -750,3 +759,256 @@ func (s *Schema) Apply(diffs []EntityDiff) (*Schema, error) {
 	}
 	return dup, nil
 }
+
+func (s *Schema) ValidateViewReferences() error {
+	var errs error
+	availableColumns := tablesColumnsMap{}
+
+	for _, e := range s.Entities() {
+		entityColumns, err := s.getEntityColumnNames(e.Name(), availableColumns)
+		if err != nil {
+			errs = multierr.Append(errs, err)
```

Comment:
I think it's safer here to `continue` this loop? So we don't setup the available columns since there's none for this table?

Reply:
agreed, done!